pyro/distributions/transforms/lower_cholesky_affine.py

```diff
@@ -0,0 +1,58 @@
+import torch
+from torch.distributions import constraints
+from torch.distributions.transforms import Transform
+
+from pyro.distributions.util import copy_docs_from
+
+
+@copy_docs_from(Transform)
+class LowerCholeskyAffine(Transform):
```

Comment:
In NumPyro, we also have [MultivariateAffineTransform transform](http://num.pyro.ai/en/stable/distributions.html#multivariateaffinetransform). Should we make the name consistent?

Reply:
i think my name is better : ) not all invertible matrices are LowerCholesky

Reply:
So we'll change the name in NumPyro when pyro-api is more matured. : )